release/release-notes/data-prepper.change-log-2.6.0.md

```diff
@@ -0,0 +1,1851 @@
+
+* __Add 2.6.0 change log__
```

Comment:
The change log hasn't been added yet. I'm not sure where it came from, but should this commit be here?